src/services/link/index.js

```diff
@@ -3,6 +3,7 @@ const service = require('feathers-mongoose');
 const logger = require('../../logger');
 const link = require('./link-model');
 const hooks = require('./hooks');
+const {REGISTRATIONLINK_EXPIRATION_DAYS} = require('../../../config/globals')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=3811219)